docs/resources/pf9_cluster.md

```diff
@@ -0,0 +1,86 @@
+- **master_sku** (String) Flavor of master nodes. Example: "Standard_A4_v2"
+- **worker_sku** (String) Flavor of worker nodes. Example: "Standard_A4_v2"
+
+### Optional
+
+- **external_dns_name** (String) Provide DNS name. For AWS this will be an A record in the provided hosted zone, routing to the cluster's master instance(s) ELB. Example: `domain.com`
+- **service_fqdn** (String) Provide FQDN for service endpoints. For AWS this will be an A record in the provided hosted zone, routing to the cluster's worker instance(s) ELB. Example: `my-services.domain.com`
+- **privileged** (Int) Allow/disallow privileged containers. Available options: 1, [0]
+- **http_proxy** (String) (optional) Specify the HTTP proxy for this cluster. Format: `<scheme>://<username>:<password>@<host>:<port>`
+- **network_plugin** (String) Network plugin to use. Available options: [flannel], calico, canal(experimental)
+- **calico_ip_ip_mode** (String) IP-IP mode if using the calico network plugin. Available options: Always, Never, CrossSubnet (default: Always)
+- **calico_nat_outgoing** (Boolean) Enable outgoing NAT for calico nodes. Default value: true
+- **tags** (Map) Tags to apply on nodes. Format: ["key1":"value1","key2":"value2"] (key-value pairs)
+- **runtime_config** (String) Runtime config data.
+- **allow_workloads_on_master** (Int) Allow workloads on master nodes. Available options: [0], 1
+- **app_catalog_enabled** (Int) Enable/Disable Platform9 [app catalog](https://platform9.com/docs/kubernetes/application-catalog). Available options: [0], 1
+- **k8s_api_port** (String) Port on which the k8s API server listens on. Default value: 443
+- **private_subnets** (List) List of private subnets to use. Example: `10.21.0.0/16`
+- **enable_metal_lb** (Boolean) Enable/disable MetalLB. Available options: true, [false]
+- **metallb_cidr** (String) MetalLB CIDR. Example: `10.21.0.0/16`
+- **api_server_flags** (List) List of custom api server flags. Example: "--request-timeout=2m0s", "--kubelet-timeout=20s"
+- **scheduler_flags** (List) List of scheduler flags. Example: "--kube-api-burst=120"
+- **controller_manager_flags** (List) List of controller manager flags. Example: "--large-cluster-size-threshold=60"
+- **internal_elb** (Boolean) Enable or disable elastic load balancer. Available options: true, [false]
+- **enable_cas** (Boolean) Enable or disable cluster auto scaler. Available options: true, [false]
+
+### Optional if using AWS
+
+- **vpc** (String) Name of AWS VPC for nodes
+- **domain_id** (String) AWS host zone id. This is found in the AWS hosted zones listing. Example: `123445sdfgfg`
+
+### Danger zone
+
+- **is_private** (Boolean) Private cluster (for advanced users only)
+- **subnets** (List) List of subnets to use (advanced)
+- **master_vip_ipv4** (String) Virtual IP for master nodes.
```

Comment:
This does not need to be in the danger zone. This is used when folks need to create multi master clusters.